keras_nlp/layers/transformer_decoder.py

```diff
@@ -252,6 +253,8 @@ def call(
                 at which to update the `cross_attention_cache`. Usually, this is
                 either `0` (compute the entire `cross_attention_cache`), or
                 `None` (reuse a previously computed `cross_attention_cache`).
+             use_causal_mask: bool, defaults to True. If true, causal mask
```

Comment:
backticks for python constants, add the missing "a" before causal mask...

```
use_causal_mask: bool, defaults to `True`. If `True`, a causal mask
```

Reply:
thanks, done